crates/ide-assists/src/handlers/convert_comment_block.rs

```diff
@@ -159,15 +163,15 @@ pub(crate) fn relevant_line_comments(comment: &ast::Comment) -> Vec<Comment> {
 //              */
 //
 // But since such comments aren't idiomatic we're okay with this.
-pub(crate) fn line_comment_text(indentation: IndentLevel, comm: ast::Comment) -> String {
-    let contents_without_prefix = comm.text().strip_prefix(comm.prefix()).unwrap();
+pub(crate) fn line_comment_text(indentation: IndentLevel, comm: ast::Comment) -> Option<String> {
+    let contents_without_prefix = comm.text().strip_prefix(comm.prefix())?;
```

Comment:
This can in theory never happen, so we can change this to
```suggestion
    let text = comm.text();
    let contents_without_prefix = text.strip_prefix(comm.prefix()).unwrap_or(text);
```
Then we don't need the other changes and it stays lazy